e2e/pages/adf/content_services/search/search-categories.ts

```diff
@@ -30,6 +31,10 @@ export class SearchCategoriesPage {
         return new SearchTextPage(filter);
     }
 
+    typeFiltersPage(filter) {
```

Comment:
Please rename typeFiltersPage method name to something general for radio widget, e.g. radioFiltersPage